numba/tests/test_dictimpl.py

```diff
@@ -181,6 +186,80 @@ def __next__(self):
     next = __next__    # needed for py2 only
 
 
+############################################################
+# Supporting code for TestDictImpl.test_parametrized_types
+############################################################
+class Parametrized(tuple):
+    def __init__(self, tup):
+        assert all(isinstance(v, str) for v in tup)
+
+
+class ParametrisedType(types.Type):
```

Comment:
Parameterize/Parameterised, am unconcerned about which spelling, but would be good to be consistent :)

Reply:
I'll try to stick with American english then ...